src/stories/containers/Finances/components/SeccionPages/DelegateExpenseTrendFinances/DelegateExpenseTrendFinances.stories.tsx

```diff
@@ -0,0 +1,106 @@
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import DelegateExpenseTrendFinances from './DelegateExpenseTrendFinances';
+import type { ComponentMeta } from '@storybook/react';
+import type { FigmaParams } from 'storybook-addon-figma-comparator/dist/ts/types';
+
+export default {
+  title: 'Components/NewFinances/Section/DelegateExpenseTrendFinances',
+  component: DelegateExpenseTrendFinances,
+
+  parameters: {
+    chromatic: {
+      viewports: [375, 834, 1194, 1440, 1920],
```

Comment:
We have integration for 1280 so we should add it here too right?

Reply:
I will add it